src/pages/components/tooltip-modal.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Tooltip Modal
+description: >
+  A Tooltip modal is a small overlay that opens automatically or through user-action, and displays contextual information for its paired element.
+package: '@zendeskgarden/react-modals'
+components:
+  - modals/src/elements/TooltipModal/TooltipModal.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To provide actions within a tooltip</li>
+      <li>To provide a focus jail when actions need to be inside a tooltip</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To reveal read-only helpful information about an element on hover, use a
+          [Tooltip](/components/tooltip)
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A Tooltip Modal must contain a header and a way to exit the experience.
```

Comment:
Rephrased to not be a directive:

`A Tooltip Modal contains a header and a way to exit the experience.`

Reply:
Updated.